Timetabling.Tests/Timetabling.Tests.csproj

```diff
@@ -137,7 +138,7 @@
     <RemoveDir Directories="$(TargetDir)lib\" />
   </Target>
   <Target Name="AfterBuild">
-    <Copy Condition=" '$(OS)' == 'Windows_NT' " SourceFiles="@(LibWindows)" DestinationFiles="@(LibWindows->'$(TargetDir)lib\%(RecursiveDir)%(Filename)%(Extension)')" />
```

Comment:
Annoying that this happens, breaks the build process on Windows. Can you manually correct this?